index.bs

```diff
@@ -2701,13 +2701,13 @@ Council Deliberations</h5>
 	in case of a tie,
 	the [=W3C Council Chair=] determines the outcome.
 	However,
-	if the decision or proposal being objected to
+	if the decision or proposal is objected to
 	originated with the TAG or AB,
 	then members of that group at the time the decision or proposal was made
 	must abstain in such a vote.
 	In case of a vote,
 	if two members of a Council who share the same affiliation cast an identical ballot,
```

Comment:
This appears to suggest that if two members of a Council who share the same affiliation cast *different* ballots, then they are both counted. I don't believe that to be the intent. I believe that all members of a Council who share the same affiliation are meant to work among themselves to condense their votes to a single ballot.